src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -146,6 +152,30 @@ && isNotBlank(appellant.getAddress().getPostcode())) {
         return preSubmitCallbackResponse;
     }
 
+    private void validateAddressAndPostcode(PreSubmitCallbackResponse<SscsCaseData> response, Entity party, String partyName) {
+        String addressLine1 = party.getAddress().getLine1();
+        String postcode = party.getAddress().getPostcode();
+
+        if (isBlank(addressLine1)) {
+            response.addError("You must enter address line 1 for the " + partyName);
+        }
+
+        if (isBlank(postcode) || !isBlank(postcode) && !postcodeValidator.isValid(postcode, context)) {
+            response.addError("You must enter a valid UK postcode for the " + partyName);
+        }
+    }
+
+    private void validatingPartyAddresses(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> response) {
```

Comment:
I would put this method before the other one because it's used first, and it uses it